testnet/docker-compose.non-sgx.yml

```diff
@@ -69,5 +69,7 @@ services:
                  "--hostAddress=$P2PPUBLICADDRESS",
                  "--logPath=sys_out",
                  "--logLevel=$LOGLEVEL",
-                 "--sequencerID=$SEQUENCERID"
+                 "--sequencerID=$SEQUENCERID",
+                 "--messageBusAddress=$MSGBUSCONTRACTADDR"
```

Comment:
I'm sure it was but can you triple check that this flag was added to all docker compose files please ?